app/code/Magento/Tax/Model/Sales/Order/TaxManagement.php

```diff
@@ -10,7 +10,6 @@
 use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Tax\Api\Data\OrderTaxDetailsAppliedTaxInterfaceFactory as TaxDetailsDataObjectFactory;
 use Magento\Tax\Api\Data\OrderTaxDetailsAppliedTaxInterface as AppliedTax;
-use Magento\Tax\Model\Sales\Order\Tax;
```

Comment:
Please avoid removal of unused imports, someday we will remove them all at once with `php-cs-fixer`. Just uncheck this inspection in PhpStorm for now or whatever tool you use for code analysis.